packages/atomic/src/components/commerce/product-list/product-template-provider.ts

```diff
@@ -6,13 +6,33 @@ import {
 
 function defaultTemplate() {
```

Comment:
Makes me wonder why we bother having JSX templates 😂 

Why are we building the DOM like this?

Reply:
+1

I guess this was copied from the search use case, where the default template is literally just a result title. Here it's getting way more complex. We should adapt it to be a`.tsx` file and return a `VNode` instead.

Reply:
Changed it to a string for now, we can revisit this at a later time